spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -61,7 +70,7 @@ deployment to fail
 ```
 cf push dataflow-server --no-start -p spring-cloud-dataflow-server-cloudfoundry-{project-version}.jar
 cf bind-service dataflow-server redis
-cf bind-service dataflow-server rabbit
```

Comment:
ok...  Let me re-ask the question.  Why are we removing rabbit from the example since it is integral for streams.   In this section `Deploying the Server app on Cloud Foundry` we are not talking about task.  Unless I'm missing something.